File: Java/src/main/java/com/semantria/mapping/output/Attribute.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlElementWrapper;
import java.util.List;

public class Attribute 
{
	private String label = null;
	private Integer count = null;
	List<Mention> mentions = null;
	
	@XmlElement(name="label")
	public String getLabel(){ return label; }
	@XmlElement(name="count")
	public Integer getCount(){ return count; }
	@XmlElementWrapper(name="mentions")
	@XmlElement(name="mention")
	public List<Mention> getMentions() { return mentions; }
	
	public void setLabel(String label){ this.label = label; }
	public void setCount(Integer count){ this.count = count; }
}

File: dotNet/Semantria.Com/Mapping/Output/Attribute.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "attribute")]
    [XmlRootAttribute("attribute", Namespace = "")]
    public class Attribute
    {
        [DataMember(Name = "label")]
        [XmlElementAttribute("label")]
        public string Label { get; set; }

        [DataMember(Name = "count")]
        [XmlElementAttribute("count")]
        public int Count { get; set; }

        [DataMember(Name = "mentions")]
        [XmlArrayAttribute("mentions")]
        [XmlArrayItemAttribute("mention", typeof(CollMention))]
        public List<CollMention> Mentions { get; set; }
    }
}
